handbook/product/planning.md

```diff
@@ -6,9 +6,9 @@ Planning is a continuous process of negotiation between product and engineering.
 
 Planning requires several artifacts for communicating. This section clarifies how each artifact is used in the planning process.
 
-### OKRs
+### OKGoals
```

Comment:
```suggestion
### Goals
```

Reply:
Haha, good find. Reminds me of https://en.wikipedia.org/wiki/OK_Go.